### DISCUSSION TITLE
Xkit extension slowing down my tumblr

### DISCUSSION THREAD

**Discussion by ashysIashy** [AprilSylph/XKit-Rewritten#753] [Help Desk]
I recently switched over to firefox recently, and added the xkit-rewritten exntesion since i used it on chrome, but i've noticed/ had happened to me that when its active my tumblr will begin to slow down after awhile, making scrolling down the site harder, and firefox will give me a popup saying that tumblr is slowing down my browser. I've deactivaed xkit and my tumblr stopped working so slow, so i believe it is a problem with xkit itself causing these issues. I don't know if its because its interacting with one of my other extensions, or if something's gone wrong on my end, but i hope it can get solved, since xkit is such a good extension for tumblr.

**Comment by marcustyphoon** [comment]
Tumblr recently fixed both performance-impacting bugs I knew about that aren't in the ads code. So, especially if you have an ad-free subscription (but even if not; those bugs were fixed in either case), I would be curious to see if you still observe this.
